katello/hooks/boot/13-hiera.rb

```diff
@@ -0,0 +1,15 @@
+TUNING_SIZES = ['small', 'medium', 'large']
```

Comment:
You would set this value based on the profiles found in the directory?

Reply:
Correct. And probably also always add `default` as an option.